src/cc-index.ts

```diff
@@ -29,10 +28,7 @@ export class CCIndex {
     public defaultCCIndex : string;
 
     constructor(crawlVersion? : string) {
-        if (!crawlVersion) {
```

Comment:
Now calling the callback with a `TypeError` when neither a default CC index was supplied in the constructor, nor in the function call.